go.mod

```
@@ -9,6 +9,7 @@ require (
 	github.com/onsi/gomega v1.17.0
 	github.com/spf13/pflag v1.0.5
 	k8s.io/api v0.23.1
+	k8s.io/apiextensions-apiserver v0.23.0
```

```suggestion
	k8s.io/apiextensions-apiserver v0.23.1
```
I would bump it to the same version as the other k8s deps.